### DISCUSSION TITLE
Welcome to cascade-demo Discussions!

### DISCUSSION THREAD

**Discussion by vcrann** [CUEDOS/helix_framework#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SamJCKnox** [comment]
Did you copy-paste that message from somewhere? XD

**Reply by vcrann** [reply]
It's what GitHub puts by default when you make the discussion page
